kafka-testing/src/test/resources/kafka_servers/kafka_consumer_protobuf.properties

```diff
@@ -0,0 +1,25 @@
+# =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+#          kafka consumer properties
+# =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+group.id=consumerGroup14
+key.deserializer=org.apache.kafka.common.serialization.IntegerDeserializer
+value.deserializer=org.jsmart.zerocode.proto.serialization.PersonDeserializer
```

Comment:
while CONSUME:
This means we can only receive Person data in a scenario, which is fine for now.
Can this be more generalized? Let's have a quick discussion over Zoom...

Same question remains while PRODUCE.